website/docs/docs/scim/main.md

```diff
@@ -20,6 +20,13 @@ PactFlow users and teams can be managed via a SCIM 2.0 compatible API. For detai
 * `/scim/User` - Manage PactFlow users
 * `/scim/Groups` - Manage PactFlow teams
 
+## Authentication
+
+The SCIM API endpoints require a Pactflow bearer token. It is recommended to use a 
+[PactFlow Service Account token](/docs/user-interface/settings/users#system-accounts) for this. This service account is
```

Comment:
We usually refer to these as "System Accounts".

Reply:
Ah, yes, we do, don't we